web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -923,7 +919,7 @@ public static String getModuleWebFolder(String moduleId) {
 		
 		moduleWebFolder += moduleId;
 		
-		return moduleWebFolder.replace("/", File.separator);
+		return Paths.get(moduleWebFolder).toString();
```

Comment:
sorry but I think we need to revert this change because of my explanation above.